code/datums/wires/explosive.dm

```diff
@@ -17,21 +17,11 @@
 	return
 
 /datum/wires/explosive/chem_grenade
-	duds_number = 2
+	duds_number = 1
 	holder_type = /obj/item/grenade/chem_grenade
 	randomize = TRUE
 	var/fingerprint
 
-/datum/wires/explosive/chem_grenade/New(atom/holder)
```

Comment:
keep the wire if you're adding EMP_PROTECT_WIRES?

Reply:
Dud wires are meant to explode and were only prevented from doing so by recent changes, which this or removes, along with reverting the decision to have 2 dud wires instead of 1